rust/src/api.rs

```diff
@@ -15,6 +15,13 @@ const CRATE_VERSION: &str = env!("CARGO_PKG_VERSION");
 pub struct SvixOptions {
     pub debug: bool,
     pub server_url: Option<String>,
+    /// Timeout for HTTP requests.
+    ///
+    /// The timeout is applied from when the request starts connecting until
+    /// the response body has finished.
+    ///
+    /// By default, requests never time out.
```

Comment:
FWIW, I think we should have some sort of opinionated default timeout, even if it's absurdly long (like 60 seconds or something). It's an easy thing to forget about when making requests to an external service.

Reply:
Okay, wanted to not change behavior at the same time but can do. Python seems to use 15s by default.